packages/preview/modern-cv/0.1.0/README.md

```diff
@@ -0,0 +1,60 @@
+# Modern CV
+
+A port of the [Awesome-CV](https://github.com/posquit0/Awesome-CV) Latex resume template in [typst](https://github.com/typst/typst).
+
+## Requirements
+
+You will need the `Robot` and `Source Sans Pro` fonts installed on your system or available somewhere. You can download them from the following links:
```

Comment:
```suggestion
You will need the `Roboto` and `Source Sans Pro` fonts installed on your system or available somewhere. You can download them from the following links:
```

Reply:
If you want you can also add a hint that the web app has both fonts and that there, no further action is necessary.